app/models/movie.rb

```diff
@@ -1,2 +1,7 @@
 class Movie < ApplicationRecord
+  with_options presence: true do
+    validates :genre
+    validates :title
+    validates :url
```

Comment:
この書き方を使用されるなら，validatesもまとめた方がよいでしょう。
```suggestion
    validates :genre, :title, :url
```

Reply:
movieモデルのバリデーションの簡素化